warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/handler/shard/ShardedDataTypeHandler.java

```diff
@@ -1141,21 +1150,40 @@ protected void createShardFieldIndexColumn(RawRecordContainer event, Multimap<Bu
      *            flag for the delete count
      * @return a value
      */
-    private Value createUidArray(String uid, boolean isDeleted) {
-        
-        // Create a UID object for the Value
-        Builder uidBuilder = Uid.List.newBuilder();
-        uidBuilder.setIGNORE(false);
-        
-        if (isDeleted)
+    protected Value createUidArray(String uid, boolean isDeleted) {
+        return new Value(createUidList(uid, isDeleted).toByteArray());
+    }
+
+    /**
+     * Create a {@link Uid.List} given a uid and delete mode flag
+     *
+     * @param uid        an event uid
+     * @param deleteMode a flag
+     * @return a Uid.List
+     */
+    protected Uid.List createUidList(String uid, boolean deleteMode) {
+        Uid.List.Builder uidBuilder = Uid.List.newBuilder();
+
+        //  delete mode takes precedent over disabled uid creation
+        if (deleteMode) {
+            uidBuilder.setIGNORE(false);
             uidBuilder.setCOUNT(-1);
-        else
+            uidBuilder.addUID(uid);
```

Comment:
This should be `addREMOVEDUID()`

Reply:
Wow. Looks like it was that way before. Have we ever used this code path?

Reply:
Yea....I'm trying to do least harm here. I don't want really want to change this to `addREMOVEDUID` unless there's a really good reason to.

Reply:
I don't think the `addUID` with a count of -1 works anymore as a "delete". I believe that logic has been removed from the GlobalIndexUidAggregator. I'm assuming we don't use this code path b/c if we did, I don't believe this would work.

Reply:
To your point, you can leave what you have and we could open a separate issue to investigate that.